ckanext/canada/i18n/fr/LC_MESSAGES/ckanext-canada.po

```diff
@@ -450,13 +450,21 @@ msgstr "L’objet JSON doit contenir la clé \"en\""
 msgid "JSON object must contain \"fr\" key"
 msgstr "L’objet JSON doit contenir la clé \"fr\""
 
+#: ckanext/canada/validators.py:527
+msgid "JSON object must contain \"fr\" key"
+msgstr "L’objet JSON doit contenir la clé \"fr\""
+
```

Comment:
is this comment for the new string below? The msgid/msgstr looks like a duplication from above?